src/implementations/vanilla/components/tag/_tag.scss

```diff
@@ -47,6 +47,11 @@ $tag: null !default;
   }
 }
 
+// No wrap
```

Comment:
i understand that this was the request in the ticket, but have we analyzed the impact of it? I mean, why are we doing this? We know that this will produce overflowing in many situations, especially because the request came out of the fact that they might have very long string used as tags, in this case the risk is maximized. It is an "opt-in" functionality, so we could say it's a user mistake when this will happen, but it's worth nothing, i see this only as a problematic style with a very limited value in certain specific use cases, maybe..